readers/common/src/test/java/com/criteo/hadoop/garmadon/reader/GarmadonMessageTest.java

```diff
@@ -70,4 +71,18 @@ public void testToMapNormalizedWithPortInfo() {
 
         TestCase.assertEquals(expected, map);
     }
+
+    @Test
+    public void testToMapOnInfinityDoubleReturnMaxNegativeValue() {
+        long timestampMillis = System.currentTimeMillis();
+        EventHeaderProtos.Header header = EventHeaderProtos.Header.newBuilder().build();
+        FlinkEventProtos.OperatorEvent.Builder operatorEvent = FlinkEventProtos.OperatorEvent.newBuilder()
+            .setRecordsLagMax(Double.NEGATIVE_INFINITY);
+
+        GarmadonMessage message = new GarmadonMessage(GarmadonSerialization.TypeMarker.FLINK_OPERATOR_EVENT, timestampMillis, header, operatorEvent.build(), null);
+
+        Map<String, Object> map = message.toMap(true, true);
+
+        TestCase.assertEquals(-Double.MAX_VALUE, map.get("records_lag_max"));
```

Comment:
Use `Double.MIN_VALUE` instead

Reply:
idem

Reply:
-Double.MAX_VALUE is the correct value (as Double.MIN_VALUE is the smallest positive nonzero value of type double.